saleor/graphql/order/resolvers.py

```diff
@@ -0,0 +1,24 @@
+from django.core.exceptions import PermissionDenied
+
+from ...order import models
+from ..utils import get_node
+from .types import Order
+
+
+def resolve_orders(info):
+    user = info.context.user
+    if user.is_anonymous:
+        raise PermissionDenied('You have no permission to see this')
+    if user.get_all_permissions() & {'order.view_order', 'order.edit_order'}:
+        return models.Order.objects.all().distinct()
+    return models.Order.objects.filter(user=user).distinct()
```

Comment:
If I'm correct this queryset would be used for example on customer's account page in the storefront to show his orders, right? In the current account view we use this queryset: `user.orders.confirmed()`. We could reuse it here to be consistent.

Reply:
Correct. I will add some docstrings explaining it and take a look at this queryset.